src/core/sys/posix/signal.d

```diff
@@ -760,7 +773,7 @@ else version( CRuntime_UClibc )
         static assert(false, "Architecture not supported.");
     }
 }
-else version (linux)
+else version (CRuntime_Bionic)
```

Comment:
[I switched this to `linux` a couple years ago](https://github.com/dlang/druntime/commit/aeecc9#diff-951b22b328337c533b375267d2f927c3R476), maybe thinking it matched the kernel APIs, but now see that it doesn't. All other C runtimes that usually run on linux have their own version blocks above, so nobody other than Bionic users were likely using this.